### PR TITLE
New post: Empowering SQL developers with real time data analytics and Apache Flink

### PR DESCRIPTION
**Pull request by Olimbia23**
# Content Pull Request Checklist

Please check to make sure your content submission meets the following requirements by adding an `x` between the brackets:

- [X] The title of this pull request reflects the title of my content.
- [X] I have reviewed the [Content Review Checklist](https://github.com/build-on-aws/content/blob/main/CONTENT_REVIEW_CHECKLIST.md).
- [X] This pull request contains one piece of content (post or page).

**By submitting this pull request, I confirm I own this content, have permission to use each image included in this content, and that it will be released under the [CC BY-SA 4.0 SA License](/LICENSE).**

### CODE REVIEW

File: posts/real-time-analytics-with-flink-sql/index.md [hunk 1]
```diff
@@ -0,0 +1,389 @@
...
+
+Performing SQL queries with KDA is possible by utilizing KDA Studio Notebooks. These Notebooks are backed by Apache Zeppelin, allowing you to query data streams interactively in real-time and develop stream processing applications that use common SQL, Python, and Scala. By clicking just a few buttons, you can start a serverless notebook in the AWS Management console to query data streams and receive quick results. These notebooks provide a user-friendly interactive development experience while taking advantage of powerful capabilities powered by Apache Flink.
+
+## Table of Content
+
+| Attributes          |                                   |
+| ------------------- | -------------------------------------- |
+| ✅ AWS Level        | 200                             |
+| ⏱ Time to complete  | 45 mins - 60 mins                      |
+| 🧩 Prerequisites       | An [AWS Account](https://aws.amazon.com/resources/create-account/?sc_channel=el&sc_campaign=devopswave&sc_content=cicdetlsprkaws&sc_geo=mult&sc_country=mult&sc_outcome=acq)|
```
Comment (riferrei) [inline posts/real-time-analytics-with-flink-sql/index.md:39]: Is it possible to specify which services are necessary?

**Review by riferrei — APPROVED** [review]
Overall, LGTM

**Comment by lenadroid** [comment]
> 

Hello Olimbia! Thank you so much for working on this great post. 

It's almost ready to publish. You asked about the CSV - the best way would be to create a repository in build-on-aws organization and include your Zeppelin notebook files there (instead of the code samples directory), together with the CSV file. I sent you instructions on how to do so via Slack. Please let me know if you have any questions!

**Review by riferrei — APPROVED** [review]
LGTM

**Pull request merged**